docs/articles/expensify-classic/settings/Set-Notifications.md

```diff
@@ -4,72 +4,66 @@ description: This article is about how to troubleshoot notifications from Expens
+
+# Troubleshooting missing Expensify notifications
+
+## Issue: The email or notification is never received, and no message, banner, or additional context is provided 
+Emails can sometimes be delayed and could take up to 30-60 minutes to arrive in your inbox. If you're expecting a notification that still hasn't arrived after waiting: 
+ - Check your **Email Preferences** on the web via **Settings > Account > Preferences**. In the **Contact Preferences** section, ensure that the relevant boxes are checked for the email type you're missing.
+ - Check your email spam and trash folders, as Expensify messages might end up there inadvertently.
+ - Check to make sure you haven't unintentionally blocked Expensify emails. Allowlist the domains expensify.com, mg.expensify.com, and amazonSES.com with your email provider.
+
+## Issue: A banner that says “We’re having trouble emailing you” shows the top of your screen.
+Confirm the email address on your Expensify account is a deliverable email address, and then click the link in the banner that says "here". If successful, you will see a confirmation that your email was unblocked. 
+
+ ![ExpensifyHelp_EmailError]({{site.url}}/assets/images/ExpensifyHelp_EmailError.png){:width="100%"}
+ 
+ **If unsuccessful, you will see another error:**
+ - If the new error or SMTP message that includes a URL, navigate to that URL for further instructions. 
+ - If the new error or SMTP message that includes "mimecast.com", consult with your company's IT team.
+ - If you see a new error or SMTP message that includes "blacklist", it means your company has configured their email servers to use a third-party email reputation or blacklisting service. Consult with your company's IT team.
+   
+![ExpensifyHelp_SMTPError]({{site.url}}/assets/images/ExpensifyHelp_SMTPError.png){:width="100%"}
 
-**For Private Domains**:
+# Further troubleshooting for public domains
 
-If your organization uses a private domain, consult your IT department or IT person to ensure that the following domains are whitelisted to receive our emails: expensify.com, mg.expensify.com, and amazonSES.com. These domains are the sources of various notification emails, so make sure they aren't being blocked.
+If you are still not receiving Expensify notifications and have an email address on a public domain, you may need to add Expensify's domains expensify.com and mg.expensify.com and amazonSES.com to your email's allowlist by taking the following steps: 
 
-**For Public Domains (e.g., Gmail, Yahoo, Hotmail)**:
+ - Search for messages from expensify.com in your spam folder, opening them, and clicking “Not Spam” at the top of the message.
+ - Configure an email filter that identifies Expensify's domains and directs all incoming messages to your inbox, to avoid messages going to spam.
```

Comment:
```suggestion
 - Configure an email filter that identifies Expensify's email domain expensify.com and directs all incoming messages to your inbox, to avoid messages going to spam.
```